server/src/main/java/edu/jhu/server/Player.java

```diff
@@ -83,6 +84,7 @@ public void sendEvent(JSONObject event) {
   @Override
   public void onWebSocketClose(int statusCode, String reason) {
     log("WebSocket closed.");
+    playerHolder.removePlayer(this);
```

Comment:
thinking out loud... if playerHolder is a game this might break things, but as we've discussed the game is probably broken if someone leaves anyway